spec/models/build_spec.rb

```diff
@@ -0,0 +1,46 @@
+require 'rails_helper'
+
+RSpec.describe Build, type: :model do
+  let(:payload) { File.read('spec/support/payloads/pull_request_opened.json') }
+  let(:build) { described_class.from_webhook(payload) }
+
+  describe '.from_webhook' do
```

Comment:
Should the call to `from_webhook ` be only scoped on this `describe` ? because if that doesnt work, the next describes will fail too.

Also I am not sure what to suggest instead. =/